Could not require 'ember-cli-build.js': Cannot find module 'broccoli-tslinter'

When I bump ember's `glimmer-engine` version to any SHA after 0fdf041f90308825e5d496ffe5c1050e5a0f2e82 I get the following error upon running `npm start` in the ember.js repo: 

```
$ npm start

> ember@2.7.0-canary start /home/zmulgrew/workspace/zth-ember.js
> ember serve

DEPRECATION: Overriding init without calling this._super is deprecated. Please call `this._super.init && this._super.init.apply(this, arguments);` addon: `ember-cli-dependency-checker`
    at Function.Addon.lookup (/home/zmulgrew/workspace/zth-ember.js/node_modules/ember-cli/lib/models/addon.js:896:27)
Could not start watchman; falling back to NodeWatcher for file system events.
Visit http://ember-cli.com/user-guide/#watchman for more info.
Could not require 'ember-cli-build.js': Cannot find module 'broccoli-tslinter'
Error: Could not require 'ember-cli-build.js': Cannot find module 'broccoli-tslinter'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/zmulgrew/workspace/zth-ember.js/node_modules/glimmer-engine/ember-cli-build.js:11:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/zmulgrew/workspace/zth-ember.js/ember-cli-build.js:45:21)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
```

I believe this is because `broccoli-tslinter` is listed as a `devDependency` in glimmer, but the ember.js `ember-cli-build.js` requires `glimmer-engine`'s `ember-cli-build.js`:

```
var glimmerEngine = require('glimmer-engine/ember-cli-build')();
```

Not  sure what the best way to fix this is? I can think of two options:

1. Move `broccoli-tslinter` to `dependencies` in `glimmer`.
2. Add `broccoli-tslinter-` to ember.js's `devDependencies` so it will "be there" after `npm install`.

Ideas?

@chadhietala @GavinJoyce

--- a/package.json
+++ b/package.json
@@ -29,6 +29,7 @@
     "broccoli-funnel": "^1.0.1",
     "broccoli-merge-trees": "^1.1.1",
     "broccoli-stew": "^1.2.0",
+    "broccoli-tslinter": "^1.0.1",
     "broccoli-typescript-compiler": "^0.6.0",
     "broccoli-uglify-js": "~0.1.3",
     "emberjs-build": "^0.10.0",
@@ -42,7 +43,6 @@
   "devDependencies": {
     "benchmark": "^1.0.0",
     "bower": "~1.3.3",
-    "broccoli-tslinter": "^1.0.1",
     "ember-cli": "^2.4.2",
     "ember-cli-release": "^1.0.0-beta.1",
     "ember-cli-sauce": "^1.3.0",

